qiskit_nature/problems/sampling/protein_folding/peptide/beads/main_bead.py

```diff
@@ -0,0 +1,70 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""A class defining a main bead of a peptide."""
+from typing import List
+
+from qiskit.opflow import PauliOp
+
+from ...peptide.beads.base_bead import BaseBead
+from ...peptide.chains.side_chain import SideChain
+
+
+class MainBead(BaseBead):
+    """A class defining a main bead of a peptide."""
+
+    def __init__(
+        self, main_index: int, residue_type: str, turn_qubits: List[PauliOp], side_chain: SideChain
+    ):
```

Comment:
I see another init missing docstring - rather than have a bunch more of these potentially this is the last one I will comment on, but evidently any place that is missing the docstring needs to be corrected